projects/extension/src/background/index.ts

```diff
@@ -305,6 +345,8 @@ chrome.runtime.onConnect.addListener((port) => {
 
         if (message.type === "chains-status-changed") {
           notifyAllChainsChangedListeners()
+        } else if (message.type === "database-content") {
+          // We never ask for the database content of a chain added through a port.
         } else {
```

Comment:
If there are no actions in `database-content` better remove `else` and just check if the type is NOT `database-content`
```suggestion
        } else if (message.type !== "database-content") {
```

Reply:
> Well, right now it's explained why the `database-content` messages are ignored

Still.. An empty condition is just not nice :)

Reply:
It does not make sense to have empty conditions. It does make the code more self-explanatory and readable - that is true - but it is "nice" not to have empty condition statements and condition check only what have an outcome based on condition.
Again though, that is opinion-based and details only... You can proceed as you find it best